sdk/extensions/wcf/Microsoft.CoreWCF.Azure.StorageQueues/src/Microsoft.CoreWCF.Azure.StorageQueues.csproj

```diff
@@ -1,44 +1,50 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <RequiredTargetFrameworks>netstandard2.0</RequiredTargetFrameworks>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <PackageIcon>Icon.png</PackageIcon>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <AssemblyName>Microsoft.CoreWCF.Azure.StorageQueues</AssemblyName>
     <PackageId>Microsoft.CoreWCF.Azure.StorageQueues</PackageId>
     <RootNamespace />
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
-    <Version>1.0.0-beta.0</Version>
+    <Version>1.0.0-preview</Version>
```

Comment:
We don't generally use `-preview` prerelease tags and doing so will likely cause other issues with our versioning automation. Any reason this cannot be a beta?